vector_search/tasks.py

```diff
@@ -271,10 +271,10 @@ def embed_new_learning_resources(self):
 @app.task(bind=True)
 def embed_new_content_files(self):
     """
-    Embed new content files from the last day
+    Embed new content files from 40 minutes ago
     """
     log.info("Running content file embedding task")
-    delta = datetime.timedelta(days=1)
+    delta = datetime.timedelta(minutes=40)
```

Comment:
This works but it's not great because we need to remember to change it again if the celery beat schedule changes . The celery beat task is called "daily_embed_new_content_files" so i think this task being updated to run every 30 minutes instead of daily is how this error occurred in the first place (things moved around enough that i'm having trouble confirming from git blame). Can you create some setting variables for  CONTENT_FILE_EMBED_FREQUNCY_MINUETES and LEARNING_RESOURCES_EMBED_FREQUENCY_MINUERES and set the value here to CONTENT_FILE_EMBED_FREQUNCY_MINUETES + 10?